ha/setup/create_pacemaker_resources.py

```diff
@@ -133,11 +133,12 @@ def sspl(cib_xml, push=False):
 
 def io_stack(cib_xml, push=False):
     """Create IO stack related resources."""
-    resources = [motr_hax, s3auth, haproxy, s3servers, s3bp]
+    resources = [motr_hax, s3auth, haproxy, s3servers]
     for rcs in resources:
         rcs(cib_xml, push)
 
     cmd_clone_group = f"pcs -f {cib_xml} resource clone io_group"
+    s3bp(cib_xml, push)
```

Comment:
Should be below SimpleProcess?

Reply:
it is simple funtion call

replaced with 
`        rcs(cib_xml, push)` line 138

Reply:
I mean, it should be called after io_group is created at line 142.

Reply:
oh missed it

Reply:
Done